Testing-jobs-and-other-features-on-production.md

```diff
@@ -20,28 +20,32 @@ All jobs must adhere to the following requirements before they can be run on the
+  - What the server jobs admin should do if the job run fails midway, both in terms of how to recover from the run (e.g. by running a job to delete all instances of a newly-created model class, etc.) and how to debug the job (e.g. do other jobs need to be run to provide additional information, is there error output that can be manually investigated, etc.)
 
 ### Submitting a PR with a new job or feature that requires third-party API
 
-**If a PR introduces a new job(s) or feature(s) that requires third-party API, it can be merged only after it is tested on the backup server.** The author of the PR should submit a request using [this form](https://goo.gl/forms/XIj00RJ2h5L55XzU2) as soon as the main reviewers seem happy with the job that it implements. (This doesn't mean that you need LGTMs from everyone, but the PR should be at the stage where only trivial, non-job-related comments are remaining.) Please still submit just a single request if multiple jobs need to be run in succession. The form responses should go to the server jobs admin (currently **@vojtechjelinek**) and the admin should run the job(s) and respond to the contributor within 48 hours. The server jobs admin should send updates after running the job(s) to the job author and [oppia release team](oppia-release-team@googlegroups.com). If the job(s) runs successfully, the admin should approve the job(s) for release in the corresponding [spreadsheet row](https://docs.google.com/spreadsheets/d/1Wegd0rZhVOm3Q3VCIw0xMbLC7IWtRyrEahiPn61Fhoo/edit#gid=948463314&range=S:S) as well as merge the PR which introduces the job. 
+**If a PR introduces a new job(s) or feature(s) that requires third-party API, it can be merged only after it is tested on the backup server.** The author of the PR should submit a request using [this form](https://goo.gl/forms/XIj00RJ2h5L55XzU2) after the job is approved by all reviewers except the server admins (admins will only approve after the job or feature is verified to be working). 
+> **Note**
+> Please submit just a single request if multiple jobs need to be run in succession. 
 
-Before submitting a testing request, please ensure that the job has already been tested manually on your local machine, and that all CI checks are passing in your PR on GitHub. For an existing job, make sure that you test the job locally and specify clearly, in the form, why you want to run the job.
+The form responses go to the server jobs admins and the admins should respond to the contributor within 24 hours. The server jobs admin should send updates after running the job(s) to the job author and [oppia release team](oppia-release-team@googlegroups.com). If the job(s) runs successfully, the admin should approve the job(s)  in the corresponding [spreadsheet row](https://docs.google.com/spreadsheets/d/1Wegd0rZhVOm3Q3VCIw0xMbLC7IWtRyrEahiPn61Fhoo/edit#gid=948463314&range=S:S) as well as merge the PR which introduces the job. 
```

Comment:
should send --> will send
should approve --> will approve
Delete extra space between "job(s)" and "in".